packages/core/src/components/date-time-card/date-time-card.tsx

```diff
@@ -20,10 +20,8 @@ export type DateTimeCardCorners = 'rounded' | 'left' | 'right';
   shadow: true,
 })
 export class DateTimeCard {
-  /**
-   * set styles
-   */
-  @Prop() individual: boolean = true;
```

Comment:
If we change/remove individual prop here we got a breaking change and cannot merge this pr as part of  upcoming minor release of v2

Reply:
> If we change/remove individual prop here we got a breaking change and cannot merge this pr as part of upcoming minor release of v2

I have changed up the logic a a little bit, if standAloneAppereance is set, it will be used - otherwise @Prop() will be used, i guess this will not produce a breaking change